templates/base.html

```diff
@@ -19,42 +19,70 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
+     <div class="off-canvas-wrapper">
+        <div class="off-canvas-wrapper-inner" data-off-canvas-wrapper>
+            <div class="off-canvas position-left" id="offCanvas" data-off-canvas>
+            <!-- Close button -->
+            <button class="close-button" aria-label="Close menu" type="button" data-close>
+            <span aria-hidden="true">&times;</span>
+            </button>
+            <!-- Menu -->
+            <ul class="vertical menu">
+                <li><a class="nav-color" href="{% url 'home' %}">HOME</a></li>
+                <li><a class="nav-color" href="{% url 'artists' %}">ARTISTS</a></li>
+                {% if user.is_authenticated %}
+                    <li style="float: center;"><a class="nav-color" href="{% url 'profile' %}">PROFILE</a></li>
+                    <li style="float: center;"><a class="nav-color" href="{% url 'logout' %}">LOGOUT</a></li>
+                {% else %}
+                    <li style="float: center;"><a data-open="login-modal">LOGIN</a></li>
+                {% endif %}
+            </ul>
+            </div>
+      <div class="off-canvas-content" data-off-canvas-content>
         <div class="full-page">
             <!-- Nav -->
+            <div class="show-for-small-only">
             <ul class="nav">
```

Comment:
It looks like here you two similar `<ul>`s. I would suggest putting the `<ul>` on the outside so that it looks like this:
```
<ul class="nav">
    <div class="show-for-small-only">
        <li><button type="button" class="menu-icon" data-toggle="offCanvas"></button></li>
    </div>
    <div class="hide-for-small-only">
        <li><a class="active" href="{% url 'home' %}">HOME</a></li>
        ...
    </div>
</ul>
```

And that one you only need one `<ul>`.